documentation/Learn-Beton/Getting-Started/Setting-up-a-local-test-server.md

```diff
@@ -17,7 +17,8 @@ change plugin configurations as you wish etc.
 Minecraft. We are going to install Paper (an improved version of Spigot) on your computer. 
 Head over to <a href="https://papermc.io/downloads" target="_blank">Papers download page</a> and click on the button you
 see in the image below (the numbers will not match up, ignore that).
-<center>![PaperMCDownload](../../media/content/LearnBeton/papermc.png)</center>
+     
+![PaperMCDownload](../../media/content/LearnBeton/papermc.png)
```

Comment:
Shouldn't this still be centered?

Reply:
or indented like the rest?

Reply:
It is automatically centered due to its format.
My centering method also does not work there 😓

Reply:
Well works after a fix.